packages/legacy/core/App/screens/NameWallet.tsx

```diff
@@ -83,19 +83,17 @@ const NameWallet: React.FC = () => {
       })
     } else {
       setLoading(true)
+
       dispatch({
         type: DispatchAction.UPDATE_WALLET_NAME,
         payload: [walletName],
       })
+
       if (agent) {
         agent.config.label = walletName
       }
+
       dispatch({ type: DispatchAction.DID_NAME_WALLET })
-      if (store.onboarding.didCompleteOnboarding) {
-        navigation.goBack()
-      } else {
```

Comment:
I think this screen can be accessed from the settings menu, not just onboarding. So that call to `navigation.goBack()` might still be needed in that case

Reply:
Fixed.